kyaml/filesys/fsondisk_test_unix.go

```diff
@@ -0,0 +1,14 @@
+// Copyright 2022 The Kubernetes Authors.
+// SPDX-License-Identifier: Apache-2.0
+
+//go:build !windows
+// +build !windows
+
+package filesys
+
+import "testing"
+
+func getOSRoot(t *testing.T) string {
+	t.Helper()
+	return "/"
```

Comment:
nit: suggest using `filepath.Separator` here. It's the same thing (`"/"`), but delegates the responsibility to Go to handle it for different OS's